examples/hosts/app-integration/external-data/src/mock-customer-service/service.ts

```diff
@@ -163,6 +190,58 @@ export async function initializeCustomerService(props: ServiceProps): Promise<Se
 		}
 	});
 
+	/**
+	 * "Echoes" the external data services data update notifications to our own webhook subscribers.
+	 *
+	 * Expected input data format:
+	 *
+	 * ```json
+	 *	{
+	 *		taskList: {
+	 *			[id: string]: {
+	 *				name: string,
+	 *				priority: number
+	 *			}
+	 *		}
+	 *	}
+	 * ```
+	 *
+	 * This data will be forwarded to our own subscribers.
+	 */
+	expressApp.post("/register-session-url", (request, result) => {
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+		const containerUrl = request.body?.containerUrl as string;
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+		const externalTaskListId = request.body?.externalTaskListId as string;
+		if (containerUrl === undefined) {
+			const errorMessage =
+				'No session data provided by client. Expected under "sessionUrl" property.';
+			console.error(formatLogMessage(errorMessage));
```

Comment:
Nit: I'd probably not log an issue for a bad request, the 400 response to the client is enough.